src/copy-filter.js

```diff
@@ -8,8 +8,8 @@ const prune = require('./prune')
 const targets = require('./targets')
 
 const DEFAULT_IGNORES = [
-  'package-lock\\.json$',
-  'yarn\\.lock$',
+  '\bpackage-lock\\.json$',
```

Comment:
```suggestion
  '(/|\b)package-lock\\.json$',
  '(/|\b)yarn\\.lock$',
```

This would probably cover all cases?

Reply:
I think adding the slash is redundant. `\b` looks for boundaries, of which `/` creates one.